packages/bundler/src/modules/ValidationManager.ts

```diff
@@ -192,6 +194,10 @@ export class ValidationManager {
     requireCond(res.aggregatorInfo == null,
       'Currently not supporting aggregator',
       ValidationErrors.UnsupportedSignatureAggregator)
+    
+    requireCond(!res.returnInfo.sigFailed,
```

Comment:
exactly.
placement doesn't really matter, but please move it to be the first `requireCond`, where it deserve to be.

Reply:
@drortirosh updated